src/components/menu-surface/menu-surface.scss

```diff
@@ -11,6 +16,8 @@
 @include menu.core-styles;
 
 .mdc-menu-surface {
+    display: var(--limel-menu-surface-display, block);
+    flex-direction: var(--limel-menu-surface-flex-direction, row);
```

Comment:
Hmm… I'm not sure I'm thinking correctly about this, but doesn't `flex-direction: row` mean that items get placed in a row, after each other, rather than in a list from top to bottom? 🤔

`row` doesn't really make sense as a default in my head…

The commit message says this just adds new css vars, but the `display` and `flex-direction` attributes weren't set on this element before. Was the element somehow already defaulting to `display: block` and `flex-direction: row`?

Reply:
The `.mdc-menu-surface` is a div and it's default display is block, so this wouldn't change in this case.

In this component `column` maybe makes more sense to use as default, but we used `row` as default because that is the default value when you use `display: flex`